.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- Thank you for your pull request to AMReX!
+
+  1. Be sure the title above is descriptive
+  2. Be sure the PR targets the development branch
+  3. Summarize the changes in a form suitable for use in case the commit is "squashed"
```

Comment:
We always squash merge.  So maybe this can be something like
```
Summarize the changes in a form suitable for use in case the combined commit messages are not suitable.
```
Or maybe we should ask the PR submitter to always provide a message that will be used as the final commit message.

Reply:
This are browser extensions for this.  https://github.com/zachwhaley/squashed-merge-message

Reply:
I tried the Firefox extension.  It works.  So I wonder if we should simply use the description (including the checklist) as the final commit message.